### PR TITLE
Multipass fixed

### PR DESCRIPTION
**Pull request by amaurea**
Looks like I forgot to make a submit this pull request, which adds multipass mapmaking support.

### CODE REVIEW

**Comment by amaurea** [comment]
The changes have functions for subtracting an existing signal estimate and for resampling degrees of freedom from a different downsample level.

File: sotodlib/mapmaking/ml_mapmaker.py [hunk 1]
```diff
@@ -346,3 +425,38 @@ def write(self, prefix, tag, m):
...
+
+    def translate(self, other, junk):
+        """Translate junk degrees of freedom from one SignalCut representation
+        to another, e.g. changing sample rate. Returns the result"""
+        self._checkcompat(other)
+        res = np.full(self.off, -1e10, self.dtype)
+        for id in self.data:
+            sdata = self .data[id]
+            odata = other.data[id]
+            so3g.translate_cuts(odata.pcut.cuts, sdata.pcut.cuts, sdata.pcut.model, sdata.pcut.params, junk[odata.i1:odata.i2], res[sdata.i1:sdata.i2])
```
Comment (iparask) [inline sotodlib/mapmaking/ml_mapmaker.py:441]: @amaurea `so3g` is not imported.

Comment (iparask) [reply sotodlib/mapmaking/ml_mapmaker.py:441]: I see. `star` imports confuse linters and I thought it was not imported. If it works, it's okay.

**Review by mhasself — APPROVED** [review]
Thanks -- Noted on the  _get_proj issue.

**Pull request merged**